### PR TITLE
Update Rubcop to v0.84

### PR DESCRIPTION
**Pull request by johnpaulashenfelter**
## Description of change

Update gem, regenerate config, and autofix where possible.
## Original issue(s)
department-of-veterans-affairs/va.gov-team#131

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -17,18 +17,26 @@ Layout/EmptyLineAfterGuardClause:
...
     - 'lib/common/models/collection.rb'
 
-# Offense count: 7
+# Offense count: 6
 # Cop supports --auto-correct.
+# Configuration parameters: AllowAliasSyntax, AllowedMethods.
+# AllowedMethods: alias_method, public, protected, private
 Layout/EmptyLinesAroundAttributeAccessor:
   Exclude:
-    - 'lib/common/models/collection.rb'
```
Comment (LindseySaari) [inline .rubocop_todo.yml:24]: Did you mean to remove this file?

Comment (johnpaulashenfelter) [reply .rubocop_todo.yml:24]: Yep -- that was fixed at some point and rubocop just picked it up :)

File: Gemfile.lock [hunk 2]
```diff
@@ -690,6 +695,13 @@ GEM
       rack (< 2.1.0)
       rack-protection (>= 1.5.0)
       redis (>= 3.3.5, < 5)
+    sidekiq-ent (1.8.1)
+      einhorn (= 0.7.4)
+      sidekiq (>= 5.2.3)
```
Comment (LindseySaari) [inline Gemfile.lock:700]: Is the sidekiq stuff here a dup of: https://github.com/department-of-veterans-affairs/vets-api/pull/4327/files

File: .rubocop_todo.yml (hunk 1 above)
Comment (LindseySaari) [reply .rubocop_todo.yml:24]: Oh cool 😎

File: Gemfile.lock (hunk 2 above)
Comment (johnpaulashenfelter) [reply Gemfile.lock]: Rebased and all right now i think

**Pull request merged**